ui-tests/cypress/support/pageObjects/app/AdminPage.js

```diff
@@ -0,0 +1,84 @@
+import {htmlElements, WebElement} from '../WebElement.js';
```

Comment:
Are you able to extract the common functionalities of _AppPage_ and this _AdminPage_ to a parent _abstract_ class to be extended by them?